src/rt/minfo.d

```diff
@@ -287,7 +287,7 @@ struct ModuleGroup
             else
                 enum EOL = "\n";
 
-            sink("Cyclic dependency between module ");
+            sink("Cyclic dependency between module ctors/dtors of ");
```

Comment:
Can we please spell out the full words.

Reply:
Done, thanks!

Reply:
Well, now we got two commits for no good reason :(

Reply:
Haha, @quickfur fixed it :) I just edited it in github. I guess I was a tad quicker.